app/views/LocationTracking.js

```diff
@@ -111,6 +184,21 @@ class LocationTracking extends Component {
     this.props.navigation.navigate('LicensesScreen', {});
   }
 
+  settings() {
+    this.props.navigation.navigate('SettingsScreen', {});
+  }
+
+  willParticipate = () => {
```

Comment:
duplicate willParticipate, this is possible a merge conflict.. please sync, and remove if still there

Reply:
This is not changed by me. Maybe an issue with some space.